CTOR 1176: Enhance FreeBSD storage - Add default ZFS support (#3821)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-freebsd-snmp.md

| Disk-Generic-Name    | OS-FreeBSD-Disk-Generic-Name-SNMP-custom    | Contrôle du taux d'espace libre disponible du disque. Pour chaque contrôle apparaîtra le nom du disque comme "label"                             |            |
| Disk-Global          | OS-FreeBSD-Disk-Global-SNMP-custom          | Contrôle du taux d'espace libre disponible des disques. Pour chaque contrôle apparaîtra le nom du disque comme "label"                           | X          |
| Disk-IO              | OS-FreeBSD-Disk-IO-SNMP-custom              | Contrôle les accès disque du disque. Pour chaque contrôle, le nom du disque apparaîtra                                                           |            |
| Inodes-Global        | OS-Freebsd-Inodes-Global-SNMP-custom        | Contrôle l'utilisation des inodes                                                                                                                |            |
| Process-Generic      | OS-FreeBSD-Process-Generic-SNMP-custom      | Contrôle permettant de vérifier le fonctionnement d'un processus/service Unix                                                                    |            |

> | Traffic-Generic-Id   | OS-FreeBSD-Traffic-Generic-Id-SNMP-custom   | Contrôle de la bande passante des interfaces. Pour chaque contrôle apparaîtra le nom de l'interface as a "label" raccourci décrivant l'interface |            |

Reviewer: as a label raccourci?

> | Traffic-Generic-Name | OS-FreeBSD-Traffic-Generic-Name-SNMP-custom | Contrôle de la bande passante des interfaces. Pour chaque contrôle apparaîtra le nom de l'interface as a "label" raccourci décrivant l'interface |            |

Reviewer: as a label raccourci?

> | Traffic-Global       | OS-FreeBSD-Traffic-Global-SNMP-custom       | Contrôle de la bande passante des interfaces. Pour chaque contrôle apparaîtra le nom de l'interface as a "label" raccourci décrivant l'interface | X          |

Reviewer: as a label raccourci?

|:---------------------|:--------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------|:----------:|
| Disk-Generic-Id      | OS-FreeBSD-Disk-Generic-Id-SNMP-custom      | Contrôle du taux d'espace libre disponible du disque. Pour chaque contrôle apparaîtra le nom du disque                   |            |
| Disk-Generic-Name    | OS-FreeBSD-Disk-Generic-Name-SNMP-custom    | Contrôle du taux d'espace libre disponible du disque. Pour chaque contrôle apparaîtra le point de montage du disque      |            |
| Disk-Global          | OS-FreeBSD-Disk-Global-SNMP-custom          | Contrôle du taux d'espace libre disponible des disques. Pour chaque contrôle apparaîtra le point de montage des disques  | X          |

> | Disk-IO              | OS-FreeBSD-Disk-IO-SNMP-custom              | Contrôle les accès disques du disque. Pour chaque contrôle apparaîtra le nom du disque                                   |            |

Reviewer: ```suggestion
| Disk-IO              | OS-FreeBSD-Disk-IO-SNMP-custom              | Contrôle les accès disque du disque. Pour chaque contrôle apparaîtra le nom du disque                                   |            |
```

</TabItem>
<TabItem value="Disk-*" label="Disk-*">

> Coming soon

Reviewer: coming soon

</TabItem>
<TabItem value="Disk-*" label="Disk-*">

| Metric name                           | Unit  |
|:--------------------------------------|:------|
| storage.partitions.count              | count |
| *disk_name*#storage.space.usage.bytes | B     |
| *disk_name*#storage.access.count      | count |

> > Applies to the following service templates: Disk-Generic-Id, Disk-Generic-Name, Disk-Global

Reviewer: EN in FR

> > To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.

Reviewer: ```suggestion
> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
```

| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). |                   |             |

</TabItem>
<TabItem value="Disk-Generic-Id" label="Disk-Generic-Id">

| Macro        | Description                                                                                                                                                                                     | Valeur par défaut                             | Obligatoire |
|:-------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------|:-----------:|
| DISKID       | Set the storage (number expected) example: 1, 2,... (empty means 'check all storage')                                                                                                           |                                               |             |

> | TRANSFORMSRC | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run'  will replace all occurrences of 'dev' with 'run' |                                               |             |

Reviewer: ```suggestion
| TRANSFORMSRC | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                                               |             |
```

> | TRANSFORMDST | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run'  will replace all occurrences of 'dev' with 'run' |                                               |             |

Reviewer: ```suggestion
| TRANSFORMDST | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                                               |             |
```

</TabItem>
<TabItem value="Disk-Generic-Name" label="Disk-Generic-Name">

| Macro        | Description                                                                                                                                                                                     | Valeur par défaut                             | Obligatoire |
|:-------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------|:-----------:|
| DISKNAME     | Set the storage (number expected) example: 1, 2,... (empty means 'check all storage')                                                                                                           |                                               |             |

> | TRANSFORMSRC | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run'  will replace all occurrences of 'dev' with 'run' |                                               |             |

Reviewer: ```suggestion
| TRANSFORMSRC | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                                               |             |
```

> | TRANSFORMDST | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run'  will replace all occurrences of 'dev' with 'run' |                                               |             |

Reviewer: ```suggestion
| TRANSFORMDST | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                                               |             |
```

| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                                              | --filter-perfdata='storage.space\|used\|free' |             |

</TabItem>
<TabItem value="Disk-Global" label="Disk-Global">

| Macro        | Description                                                                                                                                                                                     | Valeur par défaut                                       | Obligatoire |
|:-------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------|:-----------:|

> | FILTER       | Set the storage (number expected) example: 1, 2,... (empty means 'check all storage')                                                                                                           | .*                                                      |             |

Reviewer: ```suggestion
| FILTER       | Set the storage name, it can be a regex (empty means 'check all storage')                                                                                                          | .*                                                      |             |
```

> | TRANSFORMSRC | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run'  will replace all occurrences of 'dev' with 'run' |                                                         |             |

Reviewer: ```suggestion
| TRANSFORMSRC | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                                                         |             |
```

> | TRANSFORMDST | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run'  will replace all occurrences of 'dev' with 'run' |                                                         |             |

Reviewer: ```suggestion
| TRANSFORMDST | Modify the storage name displayed by using a regular expression.  Example: adding --display-transform-src='dev' --display-transform-dst='run' will replace all occurrences of 'dev' with 'run' |                                                         |             |
```

| CRITICAL     | Critical threshold                                                                                                                                                                              | 90                                                      |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                                              | --verbose --filter-perfdata='storage.space\|used\|free' |             |

</TabItem>
<TabItem value="Disk-IO" label="Disk-IO">

| Macro         | Description                                                                                        | Valeur par défaut | Obligatoire |
|:--------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | DISKNAME      | Set the device (number expected) example: 1, 2,... (empty means 'check all devices')               |                   |             |

Reviewer: storage ou disk?

|:-------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| CRITICAL     | Threshold                                                                                          | 30                |             |
| WARNING      | Threshold                                                                                          | 10                |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). |                   |             |

</TabItem>
<TabItem value="Traffic-Generic-Id" label="Traffic-Generic-Id">

| Macro        | Description                                                                                                                                            | Valeur par défaut | Obligatoire |
|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | INTERFACEID  | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored.  To filter on interface names, see --name |                   |             |

Reviewer: ```suggestion
| INTERFACEID  | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored. To filter on interface names, see --name |                   |             |
```

| CRITICALOUT  | Threshold                                                                                                                                              | 90                |             |
| WARNINGOUT   | Threshold                                                                                                                                              | 80                |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                     |                   |             |

</TabItem>
<TabItem value="Traffic-Generic-Name" label="Traffic-Generic-Name">

| Macro         | Description                                                                                                                                            | Valeur par défaut | Obligatoire |
|:--------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | INTERFACENAME | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored.  To filter on interface names, see --name |                   |             |

Reviewer: ```suggestion
| INTERFACENAME | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored. To filter on interface names, see --name |                   |             |
```

Reviewer: name ou ID?

| WARNINGOUT    | Threshold                                                                                                                                              | 80                |             |
| EXTRAOPTIONS  | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                     |                   |             |

</TabItem>
<TabItem value="Traffic-Global" label="Traffic-Global">

| Macro          | Description                                                                                                                                                                                                         | Valeur par défaut | Obligatoire |
|:---------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER         | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored.  To filter on interface names, see --name                                                              | .*                |             |

Reviewer: ```suggestion
| FILTER         | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored. To filter on interface names, see --name                                                              | .*                |             |
```
